homeassistant/components/lamarzocco/button.py

```diff
@@ -55,5 +57,14 @@ class LaMarzoccoButtonEntity(LaMarzoccoEntity, ButtonEntity):
 
     async def async_press(self) -> None:
         """Press button."""
-        await self.entity_description.press_fn(self.coordinator.device)
+        try:
+            await self.entity_description.press_fn(self.coordinator.device)
+        except RequestNotSuccessful as exc:
+            raise HomeAssistantError(
+                "Button action failed.",
```

Comment:
We can omit this one, as the translation will be used by Home Assistant at that point.

```suggestion
```

This applies to all other cases, too, of course.

Reply:
Removed it. Though I like to add the message, so it's easy to match for that specific exception in tests.

Reply:
You still can... not sure why you removed that bit?